zcash/client.go

```diff
@@ -40,7 +40,7 @@ var defaultPeerConfig = &peer.Config{
 	// TODO: fork https://github.com/gtank/btcd/blob/master/peer/peer.go
 	//       and set MinAcceptableProtocolVersion based on the most recently activated network upgrade
 	//       see ticket #10 for details
-	ProtocolVersion:  170015, // Zcash NU5 testnet (second activation)
+	ProtocolVersion: 170015, // Zcash NU5 testnet (second activation)
```

Comment:
ZIP 155 bumps the network protocol version:
https://zips.z.cash/zip-0155#deployment

Unlike Zebra, there's nothing else we need to do to support NU5 in the seeder.

```suggestion
	ProtocolVersion: 170017, // Zcash NU5 mainnet with addrv2
```